corehq/apps/reports/tests/test_project_health.py

```diff
@@ -0,0 +1,139 @@
+from django.test import TestCase
+import datetime
+from dimagi.utils.dates import add_months
+from corehq.apps.reports.standard.project_health import MonthlyPerformanceSummary
+from corehq.apps.data_analytics.models import MALTRow
+from corehq.const import MISSING_APP_ID
+from corehq.apps.domain.models import Domain
+from corehq.apps.users.models import CommCareUser
+
+
+class MonthlyPerformanceSummaryTests(TestCase):
+    DOMAIN_NAME = "test"
+    USERNAME = "testuser"
+    USERNAME1 = "testuser1"
+    USERNAME2 = "testuser2"
+    USER_TYPE = "CommCareUser"
+    now = datetime.datetime.utcnow()
+    year, month = add_months(now.year, now.month, 1)
+    month_as_date = datetime.date(year, month, 1)
+    prev_month_as_date = datetime.date(year, month - 1, 1)
+
+    @classmethod
+    def setUpClass(cls):
+        cls._setup_domain()
+        cls._setup_users()
+        cls._setup_malt_tables()
```

Comment:
Need to call super in these `setUpClass`es:
`super(MonthlyPerformanceSummaryTests, self).setUpClass()`